public/js/actions/user.js

```diff
@@ -2,12 +2,24 @@
 
 var $ = require('jquery');
 
-var appActions = require('./app');
 var actionTypes = require('constants/action-types');
+var settings = require('settings');
+
+var appActions = require('./app');
+
+
+function setUpCSRF(csrfToken) {
+  console.log('setting CSRF token for subsequent requests:', csrfToken);
+  $.ajaxSetup({
+    headers: {
+      'X-CSRFToken': csrfToken,
+    },
+  });
+}
 
 
-export function signIn(accessToken) {
-  return function(dispatch) {
+export function tokenSignIn(accessToken) {
+  return dispatch => {
```

Comment:
It's less typing though. Embrace the future! But yeah I see what you mean. Maybe we should give a try and see how bad the readability is? I think new language syntax sometimes takes getting used to. I once saw a person new to python write code like this:
````python
def foo():
    if the_thing:
        do_stuff()
    # END if
# END foo()
````
because they didn't think the indentation was readable enough for block endings.